src/components/List.js

```diff
@@ -14,9 +26,13 @@ const List = ({ token, results }) => {
         </>
       ) : null}
       <ul style={{ color: 'black' }}>
-        {results.map(result => {
-          return <li key={result.id}>{result.name}</li>;
-        })}
+        {results
+          .filter(result =>
+            result.name.toLowerCase().includes(searchTerm.toLowerCase()),
+          )
```

Comment:
Regex might be helpful to catch possessive brand names and other items with special characters.